cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -215,6 +230,7 @@ public void upgradeClusterRuntime(Set<ClusterComponent> components, boolean patc
             restartMgmtServices();
             if (patchUpgrade) {
                 downloadAndActivateParcels(products, parcelResourceApi, true);
+                callPostClouderaRuntimeUpgradeCommand(CLOUDERAMANAGER_VERSION_7_5_1, clustersResourceApi);
```

Comment:
I copied this mechanism from another method. Indeed it does not make sense to have it as a parameter however it has an advantage that if you take a look at the method call here you'll see immediately that this is only enabled since 7.5.1 and you don't have to look into the method to find this out. Due to this I would keep this method as is.